specs/spring-cloud-contract-spec-kotlin/src/main/kotlin/org/springframework/cloud/contract/spec/internal/CommonDsl.kt

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.contract.spec.internal
+
+import java.io.File
+import java.net.URISyntaxException
+import java.nio.charset.Charset
+import java.util.regex.Pattern
+
+/**
+ * @author Tim Ysewyn
+ */
+open class CommonDsl {
```

Comment:
Is `open` class similar to public scope? if yes, add javadocs with more info (alos description, `@since`, etc.) to all such classes (the public api in general).

Reply:
No, by default classes are final in Kotlin. To be able to extend a class you need to add this `open` modifier.